src/strategy/mod.rs

```diff
@@ -89,10 +89,16 @@ impl UpdateStrategy {
             UpdateStrategy::FleetLock(_) => self.configuration_label().to_string(),
             UpdateStrategy::Immediate(_) => self.configuration_label().to_string(),
             UpdateStrategy::Periodic(p) => format!(
-                "{}, total schedule length {} minutes (next window {})",
+                "{}, total schedule length {} minutes; next window {}",
                 self.configuration_label(),
                 p.schedule_length_minutes(),
-                p.human_remaining()
+                // If not using UTC, we cannot accurately calculate remaining time;
+                // display next window week day and time, instead.
+                if p.time_zone != tzfile::Tz::named("UTC").unwrap() {
```

Comment:
I think it makes sense to place this into a `p.is_calendar_utc() -> bool`, caching the boolean upfront instead of re-computing it each time.

Reply:
We now have a `p.tz_name()` that gets the "cached" name of the time zone. Using this over a boolean because we need to use this name in `human_next_reboot_window()` too.